action/sealedenvelope_test.go

```diff
@@ -144,16 +144,29 @@ func TestSealedEnvelope_Proto(t *testing.T) {
 
 	se2 := SealedEnvelope{}
 	for _, v := range []struct {
+		name     string
 		encoding iotextypes.Encoding
 		sig      []byte
-		err      string
+		err      error
 	}{
-		{0, _signByte, "invalid signature length ="},
-		{3, _validSig, "unknown encoding type"},
+		{
+			"InvalidSignature",
+			0,
+			_signByte,
+			ErrInvalidSigLen(len(_signByte)),
+		},
+		{
+			"InvalidEncodingType",
+			3,
+			_validSig,
+			ErrUnknownEncType(3),
+		},
 	} {
-		se.encoding = v.encoding
-		se.signature = v.sig
-		req.Contains(se2.LoadProto(se.Proto()).Error(), v.err)
+		t.Run(v.name, func(t *testing.T) {
+			se.encoding = v.encoding
+			se.signature = v.sig
+			req.Equal(se2.LoadProto(se.Proto()).Error(), v.err.Error())
+		})
```

Comment:
as we discussed, please revert unnecessary changes, do minimum changes to make the test pass